src/args/default.nix

```diff
@@ -72,22 +19,13 @@ let
     projectPathsMatching = import ./project-paths-matching/default.nix args;
     projectSrcInStore = builtins.path { name = "head"; path = projectSrc; };
     inherit projectSrcMutable;
-    removePrefix = lib.removePrefix;
-    securePythonWithBandit = import ./secure-python-with-bandit/default.nix args;
-    sortAscii = builtins.sort (a: b: a < b);
-    sortAsciiCaseless = builtins.sort (a: b: lib.toLower a < lib.toLower b);
-    taintTerraform = import ./taint-terraform/default.nix args;
-    testTerraform = import ./test-terraform/default.nix args;
-    toDerivationName = lib.strings.sanitizeDerivationName;
-    toBashArray = import ./to-bash-array/default.nix args;
-    toBashMap = import ./to-bash-map/default.nix args;
-    toFileJson = import ./to-file-json/default.nix args;
-    toFileToml = import ./to-file-toml/default.nix args;
-    toFileYaml = import ./to-file-yaml/default.nix args;
-    toFileJsonFromFileYaml = import ./to-file-json-from-file-yaml/default.nix args;
-    toFileLst = import ./to-file-lst/default.nix;
-  };
 
-  lib = __nixpkgs__.lib;
+    # Impure, require sandbox and restricted evaluation set to false
```

Comment:
@kamadorueda All these are impure because of node, right?

Reply:
Mostly because the derivation does npm install (network access) without  ensuring $out is stable with a hash, I'm gonna polish that soon